.github/workflows/ci.yml

```diff
@@ -528,6 +528,9 @@ jobs:
       matrix:
         box:
           - fedora/39-cloud-base
+          # the upstream rockylnux/8 image no longer works:
+          # https://github.com/containerd/containerd/pull/10297
+          - bento/rockylinux-8
```

Comment:
Still failing
https://github.com/containerd/containerd/actions/runs/9395253328/job/25885528463?pr=10304

```
==> default: Starting domain.
==> default: Domain launching with graphics connection settings...
==> default:  -- Graphics Port:      5900
==> default:  -- Graphics IP:        127.0.0.1
==> default:  -- Graphics Password:  Not defined
==> default:  -- Graphics Websocket: 5700
==> default: Waiting for domain to get an IP address...
==> default: Removing domain...
==> default: Deleting the machine folder
/root/.vagrant.d/gems/3.1.4/gems/fog-core-2.4.0/lib/fog/core/wait_for.rb:9:in `block in wait_for': The specified wait_for timeout (2 seconds) was exceeded (Fog::Errors::TimeoutError)
```
```

Reply:
Changed to Alma Linux 8